src/drivers/uavcan/uavcan_params.c

```diff
@@ -225,14 +225,14 @@ PARAM_DEFINE_INT32(UAVCAN_SUB_BARO, 0);
  * subscription battery
  *
  * Enable UAVCAN battery subscription.
- *  1) uavcan::equipment::power::BatteryInfo
- *  2) cuav::equipment::power::CBAT
+ *  uavcan::equipment::power::BatteryInfo
+ *  ardupilot::equipment::power::BatteryInfoAux
  *
  * @min 0
  * @max 2
- * @value 0 disabled
- * @value 1 default
- * @value 2 CUAV battery monitor
+ * @value 0 Disabled
+ * @value 1 BatteryInfo
+ * @value 2 BatteryInfo + BatteryInfoAux
```

Comment:
Any reason why you specifically wouldn't want BatteryInfoAux enabled? I think it would be fine as a single boolean parameter (UAVCAN_SUB_BAT).